src/common/dependency-manager/local-manager.ts

```diff
@@ -143,8 +146,8 @@ export default class LocalDependencyManager extends DependencyManager {
 
   toExternalHost(node: DependencyNode) {
```

Comment:
@tjhiggins would you want this to take an interface key as a param? Or leave as is because until we make decisions about magic params, we really only support one interface per service?

Reply:
Leaning towards option one